settings.py

```diff
@@ -116,6 +116,7 @@
 FORMPLAYER_TIMING_FILE = "%s/%s" % (FILEPATH, "formplayer.timing.log")
 FORMPLAYER_DIFF_FILE = "%s/%s" % (FILEPATH, "formplayer.diff.log")
 SOFT_ASSERTS_LOG_FILE = "%s/%s" % (FILEPATH, "soft_asserts.log")
+MAIN_COUCH_SQL_DATAMIGRATION = "%s/%s" % (FILEPATH, "main_couch_sql_datamigration.log")
```

Comment:
you may have already planned on doing this, but if the command takes a long time for some projects you'll want to run it from a setup release that uses `keep_days` so that this log file (and the code that's being run) don't get wiped away after 3 deploys

Reply:
That's a good tip. I hadn't thought that through

Wouldn't the log file at least get copied over though? That was my assumption, but maybe I should look more in depth at the deploy pipeline

Reply:
it won't get copied - it will just get deleted with the release. the only things that get copied are things that the deploy process explicitly copies over (like localsettings)